plugins/cookie_manager/test/cookies_test.dart

```diff
@@ -49,6 +49,33 @@ void main() {
       HttpHeaders.cookieHeader: mockSecondRequestCookies,
     });
 
+    cookieManager.onRequest(options, mockRequestInterceptorHandler);
+  });
+  test('testing set-cookies parsing', () async {
+    const List<String> mockResponseCookies = [
+      'key=value; expires=Sun, 19 Feb 3000 00:42:14 GMT; path=/; HttpOnly; secure; SameSite=Lax',
+      'key1=value1; expires=Sun, 19 Feb 3000 01:43:15 GMT; path=/; HttpOnly; secure; SameSite=Lax, '
```

Comment:
```suggestion
      'key1=value1; expires=Sun, 19 Feb 3000 01:43:15 GMT; path=/; HttpOnly; secure; SameSite=Lax',
```

Reply:
Right?

Reply:
Nope. This line is intended to test with multi-value parsing.

Reply:
No, its a single string split for convenience and readability in 2 lines. So no commas as separator.

Reply:
> Nope. This line is intended to test with multi-value parsing.

OKay